src/main/kotlin/com/openlattice/auditing/AuditRecordEntitySetsManager.kt

```diff
@@ -137,7 +137,8 @@ class AuditRecordEntitySetsManager(
         val firstUserPrincipal: Principal
 
         try {
-            firstUserPrincipal = ownerPrincipals.first { it.type == PrincipalType.USER }
+            firstUserPrincipal = ownerPrincipals.firstOrNull { it.type == PrincipalType.USER }
+                    ?: ownerPrincipals.first { it.type == PrincipalType.ORGANIZATION }
```

Comment:
I'm not sure we've thought through using ORGANIZATION principal as the owner. 

If anything we should be using Organization ADMIN role as the owner of the catalog role and the audit entity sets. In other words, it seems more natural to allow ROLES to be owners of objects. 

It will also cause issues down the road when roles are deleted and there are objects are left without owners.

Reply:
Oooh true. Yeah seems like admin role should be the owner here (+ build in tracking for those roles someday). We should also block deletion of admin role if we're not doing that already